src/scala/com/github/johnynek/bazel_deps/IO.scala

```diff
@@ -39,6 +46,7 @@ object IO {
    */
   case class WriteFile(f: Path, data: Eval[String]) extends Ops[Unit]
   case class Failed(err: Throwable) extends Ops[Nothing]
+  case class Compare(path: Path, data: Eval[String]) extends Ops[FileComparison]
```

Comment:
actually, can we remove this and instead add `ReadFile(path: Path) extends Ops[String]` I think that is more general and can be used to build
```scala
def compare(p: Path, data: Eval[String]): IO[FileComparison] = 
  readFile(p).map { contents => FileComparison(p, contents == data.value) }
```
We can also use `readFile` in other places where we want do pure IO.

Reply:
sure, will do

Reply:
done -- what do you think?  I had to wrap in Option to distinguish empty files from missing files.